playwright/_impl/_fetch.py

```diff
@@ -405,7 +404,8 @@ async def _inner_fetch(
             "fetch",
             {
                 "url": url,
-                "params": params_to_protocol(params),
+                "params": object_to_array(params) if isinstance(params, dict) else None,
+                "encodedParams": params if isinstance(params, str) else None,
```

Comment:
I haven't seen this change in .net or java roll for some reason. Could you please double check?

Reply:
In Java we don't allow passing query params as strings while in JS/Python/.NET [we do](https://github.com/microsoft/playwright/pull/32516). There is no .NET roll yet for this release after we moved the merge logic to the [server side](https://github.com/microsoft/playwright/pull/32518).

In Java we have:
- https://playwright.dev/java/docs/api/class-requestoptions#request-options-set-query-param
- https://github.com/microsoft/playwright-java/blob/46f4ac1f33987ee3bc086e598d6d0ba197777087/playwright/src/main/java/com/microsoft/playwright/impl/RequestOptionsImpl.java

Reply:
I'd rather not do it in the ports to keep the API surface smaller, especially if there is no user request for this.